### PR TITLE
Patch Werkzeug breaking change

### PR DESCRIPTION
**Pull request by josh-allan**
- See https://github.com/clegaspi/saml_reader/issues/85 for writeup.

### CODE REVIEW

**Comment by clegaspi** [comment]
Thanks for submitting this, Josh! Can you please add this pinned version of this package to `setup.py` in `install_requires`? I'll see about upgrading the dash framework to a more modern version to keep up with the times.

File: README.md [hunk 1]
```diff
@@ -100,20 +104,20 @@ To pull down the latest version:
...
 2. Clone the repository locally with `git clone`.
 3. Create a virtual environment such as [virtualenv](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) or [Anaconda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html) using Python 3.9+ and activate it.
 4. `brew edit libxmlsec1` and replace the entire contents of the brew package with the following [gist](https://raw.githubusercontent.com/Homebrew/homebrew-core/7f35e6ede954326a10949891af2dba47bbe1fc17/Formula/libxmlsec1.rb)
 5. `brew unlink libxmlsec1` and `brew uninstall libxmlsec1`
 6. In your shell `rc` file (`.bashrc`, `.zshrc`, etc.), set the environment variable `HOMEBREW_NO_INSTALL_FROM_API` to `1`.
 7. Force a clean installation with `brew install /opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/libxmlsec1.rb`
 8. Trigger a clean installation of the pip package with `pip install . --no-cache-dir`.
-9. Remove `HOMEBREW_NO_INSTALL_FROM_API` from your shell `rc` file.
-10. (optional) To keep `brew` from upgrading the package, run `brew pin libxmlsec1`.
+9. Downgrade the version of `Werkzeug` from `3.0.1` to `2.2.2` using `pip install Werkzeug==2.2.2`.
```
Comment (clegaspi) [inline README.md:115]: Can you copy this instruction to the Install from PyPI above and leave a note referencing the issue you opened?

**Review by clegaspi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (josh-allan) [reply README.md:116]: Done.

File: README.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ To install SAML Reader from PyPI:
 
 1. It is **highly recommended** that this package be run in a Python virtual environment such as [virtualenv](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) or [Anaconda](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html). Please follow one of the previous links to learn how to create a Python environment of your choice. Create the environment with Python 3.6-3.10 and activate it. **SAML Reader is not currently compatible with Python 3.11+.** I do not recommend installing this directly into your system's global environment. There is just so much that can go wrong.
 2. Install the package from PyPI:
+3. Downgrade Werkzeug to `2.2.2` with `pip install Werkzeug==2.2.2` as the current release version introduces a breaking change and this version negates that change until a later Web Application Framework can be utilised. This is detailed in the Github issues section for [issue 85](https://github.com/clegaspi/saml_reader/issues/85)
```
Comment (clegaspi) [inline README.md:63]: Put this on line 69 and move current line 69 to 70 and change to step 4.

File: setup.py [hunk 3]
```diff
@@ -4,28 +4,30 @@
...
+    packages=find_packages(),
+    entry_points={
+        "console_scripts": [
+            "saml_reader=saml_reader.cli:start_saml_reader",
+            "saml_web_app=saml_reader.web.cli_hook:start_web_app_from_cli",
+        ]
+    },
+    install_requires=[
+        "pyperclip",
+        "Werkzerg==2.2.2",
```
Comment (clegaspi) [inline setup.py:22]: Typo

**Review by clegaspi — CHANGES_REQUESTED** [review]

**Review by clegaspi — APPROVED** [review]
LGTM, thanks Josh

**Pull request merged**